k9mail/src/main/java/com/fsck/k9/activity/MessageList.java

```diff
@@ -1435,6 +1435,8 @@ public void enableActionBarProgress(boolean enable) {
     public void displayMessageSubject(String subject) {
         if (mDisplayMode == DisplayMode.MESSAGE_VIEW) {
             mActionBarSubject.setText(subject);
```

Comment:
What is is subject when it's not the email subject. Why are we having to do this crap? What's calling this with an empty string?